e2e-tests/orchestrator/orchestrator_test.go

```diff
@@ -1005,7 +1005,7 @@ func secureHeadersAddAppOrch() map[string][]string {
 
 	appOrchSecureHeaders["Content-Security-Policy"] = []string{fmt.Sprintf("default-src 'self'; form-action 'self'; object-src 'none'; frame-ancestors 'none'; script-src 'self' ; frame-src 'self' https://keycloak.%s; style-src 'self'; img-src 'self' data:; connect-src 'self' https://keycloak.%s; upgrade-insecure-requests; block-all-mixed-content", //nolint: lll
 		serviceDomain, serviceDomain)}
-	appOrchSecureHeaders["Cross-Origin-Embedder-Policy"] = []string{"unsafe-none"}
```

Comment:
you can just remove this line 1008 as is just an override of the original and it will pick it up from the original

Reply:
done, thanks Sean